doc/building.rst

```diff
@@ -39,7 +39,15 @@ At a minimum, you will need to specify:
     |RTI_CONNEXT| installation.
 
 Optionally, you can use ``CMAKE_BUILD_TYPE`` to select the type of libraries to
-build (either ``Release``, or ``Debug``, default: ``Debug``).
+build (either ``Release``, or ``Debug``, default: ``Debug``). This documentation
```

Comment:
"Release" and "Debug" are not the only build configuration accepted by CMake:
>  Possible values are empty, Debug, Release, RelWithDebInfo, MinSizeRel, ...

Also, the default build type is not "Debug" in CMake, but rather the "empty" configuration, which is slightly different. Originally, the old repository used to "normalize" the build type if it was empty, setting it to "Release" and thus making "Release" the default. Not sure why we lost that feature.

Anyway, I suggest a small rephrasing:
```suggestion
build (for example ``Release``, or ``Debug``, see CMake's own documentation for [CMAKE_BUILD_TYPE](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html)). This documentation
```